gui.py

```diff
@@ -234,27 +247,13 @@ def wrap_in_section(content):
     ],
 )
 
-about = html.Div(
-    className="section about content is-size-5",
-    children=[
-        dcc.Markdown(
-            """
-
-This chart compares the current year's daily tally of acres burned to all high fire years (> 1 million acres burned) since daily tally records began in 2004.  Click camera icon at upper right of diagram to download chart.
-
-"""
-        )
-    ],
-)
 
 after_chart = html.Div(
     className="section about-after content is-size-5",
     children=[
         dcc.Markdown(
             """
-
- * Daily tallies go up or down as improved estimates and data become available throughout the fire season.
- * Data provided by the [Alaska Interagency Coordination Center (AICC)](https://fire.ak.blm.gov).
+ *
```

Comment:
this little guy!

Reply:
Removed!  There was extra cruft there that was supposed to be deleted too.  Check & resolve if you think I got it.